lib/identity_config.rb

```diff
@@ -259,7 +259,6 @@ def self.build_store(config_map)
     config.add(:in_person_proofing_enabled, type: :boolean)
     config.add(:in_person_proofing_opt_in_enabled, type: :boolean)
     config.add(:in_person_public_address_search_enabled, type: :boolean)
-    config.add(:in_person_residential_address_controller_enabled, type: :boolean)
```

Comment:
Same concern re: 50/50 state. This might need to be deployed second after we remove all reads to the configuration value.